### PR TITLE
[improve](doris compose) ls detail command print node's path

### PR DESCRIPTION
**Pull request by yujun777**
## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

File: regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/SuiteCluster.groovy [hunk 1]
```diff
@@ -168,6 +178,10 @@ class Backend extends ServerNode {
         return NodeType.BE
     }
 
+    String getLogFilePath() {
```
Comment (deardeng) [inline regression-test/framework/src/main/groovy/org/apache/doris/regression/suite/SuiteCluster.groovy:181]: 要不把
conf
doris-meta
storage
都暴露出来，回归case中就好做些事了

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run p1

**Comment by yujun777** [comment]
run cloud_p0

**Comment by yujun777** [comment]
run cloud_p0

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run buildall

**Review by deardeng — APPROVED** [review]
LGTM

**Comment by yujun777** [comment]
run p0

**Comment by yujun777** [comment]
run cloud_p1

**Comment by yujun777** [comment]
run p0

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**